### ISSUE TITLE
OHMS TOC keywords populate from two fields

### ISSUE THREAD

**Issue by apinkney0696** [sciencehistory/scihist_digicoll#1954]
@rachellane12 and the oral history department did some informal user testing with Michelle and family and friends to look at the use of OHMS interviews. They are subsequently reworking their guidelines for keyword use in the table of contents and need a little help with the development. 
(Ticket 2 of 2)

**2. Pull table of contents section keywords from both keyword and subject heading input sections in the OHMS backend.** 
Currently OH staff is only using the keyword input box, which auto-fills from their local vocabulary. They would like to also use the subject input box because it auto-fills from LC subject headings. 

The hope is that any term inputted into either the keyword or subject field would appear in the "Keyword" section in the public interface.

**Comment by jrochkind** [comment]
Should subjects also... . show up as work-level subjects, including work-level facets?  I guess we can ignore that from now, just treat them the same way as keywords -- we should make sure to index them the same way we index keywords though, so they can still be found in a search! 

That applies to the weird JS "search inside the work" function we've done too.

**Comment by apinkney0696** [comment]
No, these are not work-level descriptors and shouldn't show up at that level. These are just for the individual table of contents sections. The current way that keywords are treated in search is fine!

**Comment by rachellane12** [comment]
@jrochkind, I added subjects on the OHMS backend to the first three sections for Panning: Childhood in Canada and trips to Switzerland; Attending McMaster University/Studying molecular biology/Lab research and techniques; and Cancer research group in graduate school/Requirements of graduate program/Studying herpes viruses in James R. Smiley's lab and updated the XML file in the DC.

Here's what it looks like in OHMS for one section with both subjects and keywords:
<img width="306" alt="Screenshot 2022-11-28 133009" src="https://user-images.githubusercontent.com/98541497/204353485-b7145157-d5e5-4ac7-9092-e5658e845246.png">

**Comment by rachellane12** [comment]
Obviously, right now in the DC only the keywords appear:
<img width="410" alt="Screenshot 2022-11-28 133202" src="https://user-images.githubusercontent.com/98541497/204353777-1cbd21bb-e548-4272-ba83-a83606ffaee5.png">

**Comment by jrochkind** [comment]
Got it thanks! We'll have this for you sometime this week or early next!

**Comment by rachellane12** [comment]
Good to know about subjects! I will start doing that. Thanks!

**Comment by jrochkind** [comment]
I mean, to be clear -- if you change subjects in OHMS for something *already imported into Digital Collections*, you'll have to re-import it. I'm not sure if you've ever done that kind of re-import? It can be done. 

My point was just that anything that DOES get imported into D.C. with subjects (either as the first time for newly OHMS-ed thing, or a re-import) -- it's fine to start doing that now, and the data will be waiting for when we add the feature, no problem.

**Comment by rachellane12** [comment]
Gotcha. That makes sense, Jonathan. I did do a re-import for the Panning OHMS so that we'd have one interview with some subjects, so I'm feeling good on that front!

**Comment by jrochkind** [comment]
@rachellane12 this should be live now!

I think Panning now includes the "subjects" together with "keywords" in the "keywords" field: https://digital.sciencehistory.org/works/5z7vjlx#t=0&tab=ohToc

And confirming the subjects are "searchable" too: https://digital.sciencehistory.org/catalog?f%5Bgenre_facet%5D%5B%5D=Oral+histories&q=%22Science--Study+and+teaching+%28Secondary%29%22

Let me know if there are any issues!
